docs/src/benchmarks.md

```diff
@@ -3,23 +3,57 @@ CurrentModule = Cluster
 ```
 # Benchmarks
 
-Here you will soon find benchmarks comparing the performance of the clustering algorithms in this package to other packages.
+For benchmark our project we used the dataset provided by [Marek Gagolewski](https://clustering-benchmarks.gagolewski.com/index.html). Specificallyh we are using [wut](https://clustering-benchmarks.gagolewski.com/weave/suite-v1.html#sec-battery-wut) dataset. Below you can see the table with our metrics of rand index and accuracy of our algorithms: kmeans and kmeans++ which was benchmarked on wut dataset. Also, we are providing visualization in order to give insights what data we used.
 
-```@example 1
```

Comment:
use @eval to actually run and output the benchmark here
https://documenter.juliadocs.org/stable/man/syntax/#@eval-block